tessera-core/src/main/java/com/quorum/tessera/node/PartyInfoParser.java

```diff
@@ -90,14 +91,12 @@ default PartyInfo from(final byte[] encoded) {
         //so the prefix is always 2 (2 elements) and
         final List<byte[]> recipients = partyInfo.getRecipients()
             .stream()
-            .map(r -> {
```

Comment:
This needs to be reversed for backwards compatibility.
The encode array method will add 8 bytes to the start of this field which shouldn't be there.